integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -57,20 +59,84 @@ CONFIGS().forEach(({lib, setup, protocol, txRollupAddress}) => {
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
 
       const contract = await op.contract();
-      const symbolReturn = await contract.storage()
-      expect(JSON.stringify(symbolReturn)).toEqual(JSON.stringify(UnitValue))
+      console.log(contract.address);
```

Comment:
I think this console log was left in by accident

Reply:
oop thanks